src/OpcApplicationConfiguration.cs

```diff
@@ -208,6 +210,11 @@ public async Task<ApplicationConfiguration> ConfigureAsync()
 
         // show certificate store information
         await ShowCertificateStoreInformationAsync().ConfigureAwait(false);
+        ApplicationConfiguration.ServerConfiguration.MaxSessionCount = MaxSessionCount;
+        ApplicationConfiguration.ServerConfiguration.MaxSubscriptionCount = MaxSubscriptionCount;
+        Logger.Information("Application configured with MaxSessionCount: {maxSessionCount} and MaxSubscriptionCount: {maxSubscriptionCount}",

```

Comment:
add these setting in the serverBuilder with the fluent API

Reply:
sure!